website/profile/views.py

```diff
@@ -382,6 +383,39 @@ def user_notifications(auth, **kwargs):
         'mailing_lists': auth.user.mailing_lists
     }
 
+@dev_only
+@must_be_logged_in
+def oauth_application_list(auth, **kwargs):
+    """Return app creation page with list of known apps"""
+    # TODO: Hardcoded URL
```

Comment:
You always have to reference by *something*, right? Whether it's by the view function or by the route or whatever, there still has to be a reference. In the case of API V2 routes, the only guarantee we give anyone, developers or users, that something is going to be consistent over time is that, if it's a public route, that the route is not going to change. So what's the problem with using the route this way?

Granted, this may not end up being a public route, but I'd rather not be moving around private routes a lot, either.